x/stakeibc/keeper/msg_server_submit_tx.go

```diff
@@ -252,17 +248,21 @@ func (k Keeper) SubmitTxs(
 	callbackId string,
 	callbackArgs []byte,
 ) (uint64, error) {
-	k.Logger(ctx).Info(fmt.Sprintf("SubmitTxs %v", msgs))
 	chainId, err := k.GetChainID(ctx, connectionId)
 	if err != nil {
 		return 0, err
 	}
-	owner := types.FormatICAAccountOwner(chainId, account.GetTarget())
+	owner := types.FormatICAAccountOwner(chainId, account.Target)
 	portID, err := icatypes.NewControllerPortID(owner)
 	if err != nil {
 		return 0, err
 	}
 
+	k.Logger(ctx).Info(utils.LogWithHostZone(chainId, "Submitting ICA Tx on %s, %s with TTL: %d", portID, connectionId, timeoutTimestamp))
```

Comment:
What do you think about putting an extra "    " here, and then adding a further indent on line 263? The thinking being the ICA submission is kind of a "sub-step" in a function call, so it could make the logs a bit easier to read (more clear indents for function nesting)

Reply:
Great call, adding!